VRDR.HTTP/Nightingale.cs

```diff
@@ -610,54 +647,78 @@ private static void SetStringValueDeathRecordDictionary(DeathRecord deathRecord,
             prop.SetValue(deathRecord, dict, null);
         }
 
-        /// <summary>Set a Yes/No (coded) value on a DeathRecord.</summary>
-        private static void SetYesNoValueDeathRecordCode(DeathRecord deathRecord, string property, string value)
+        /// <summary>Set a Yes/No (coded) value on a DeathRecord using Boolean helpers.</summary>
+        private static void SetYesNoValueDeathRecordBoolean(DeathRecord deathRecord, string property, string value)
```

Comment:
It seems like this function is useful for other parts of the library, should it be in VRDR vs VRDR.HTTP?

Reply:
Seems reasonable, but should only be moved when it's actually first used outside of VRDR.HTTP (YAGNI principle)